crates/subspace-farmer/docs/plotting.md

```diff
@@ -0,0 +1,49 @@
+`Plotting` structure is the abstraction on top of the plotting process on the
+single replica.
+
+Plotting Instance that stores a channel to stop/pause the background farming
+task and a handle to make it possible to wait on this background task.
+
+It does several things.
+
+### Listen for new blocks produced by network
+
+In order to plot whole blockchain history we need to receive network updates,
+for that we regularly ask node for its best block number and after that plot
+block which is under some confirmation depth constant.
+
+TODO: make plotting account for forks
+
+### Archiving blocks
+
+After that we request finalized (blocks under some confirmation depth from the
```

Comment:
Yeah, Ikr. But I elaborated what I mean by that. Finalized block is just a block which is under some confirmation depth. What would be a better term here?

Reply:
Just try to avoid words that have a specific canonical meaning in the industry and especially if they can be later used in our protocol since it can easily confuse reader.

This paragraph misleadingly uses finalization and mentions method that is called, but tells nothing about why this is even necessary.

Something like this is way more helpful:
> This is currently necessary due to implementation challenge where archiving that happens on the node is not waiting for farmer, also farmer can connect/disconnect from node at any time, thus resulting in farmer potentially missing some of the archived pieces altogether. As such, farmer temporarily has its own archiving process as well. It will eventually be replaced with DNS-based subscriptions where pieces are disseminated by executors.